src/views/OrderDetail.vue

```diff
@@ -1271,8 +1277,59 @@ export default defineComponent({
         }
       })
       modal.present();
+    },
+    canRequestTransfer(order: any): boolean {
+      return (
+        order?.shipGroup?.shipmentMethodTypeId === 'STOREPICKUP' &&
+        order?.statusId === 'ORDER_APPROVED' &&
+        !order?.shipGroup?.shipmentId &&
+        !order?.readyToHandover &&
+        !order?.readyToShip &&
+        !order?.rejected &&
+        !order?.shipGroup?.items?.some((item: any) => item.rejectReason)
+      );
+    },
+
+    async confirmRequestTransfer(order: any) {
+      const header = this.translate('Convert to Ship-to-Store');
+      const message = this.translate(
+        'This BOPIS order will be converted to Ship-to-Store. The order will be fulfilled from a warehouse and shipped to this store for customer pickup. Continue?'
+      );
+
+      const alert = await alertController.create({
+        header,
+        message,
+        buttons: [
+          { text: this.translate('Cancel'), role: 'cancel' },
+          { text: this.translate('Convert'), handler: async () => await this.requestTransfer(order) }
```

Comment:
Improve code formatting to add each property on a new line and format the handler part, you can refer already existing alerts for formatting.